<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;FreeSans&quot;;
background-color: rgb(136, 138, 133);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="calculator">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="select_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="1" colspan="2">
           <widget class="QPushButton" name="databaseButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(114, 159, 207);</string>
            </property>
            <property name="text">
             <string>БД</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>550</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);
selection-background-color: rgb(239, 41, 41)</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="credittab">
           <attribute name="title">
            <string>Кредит</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QWidget" name="credit" native="true">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0" colspan="3">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="creditSaveButton">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(201, 94, 22);</string>
                   </property>
                   <property name="text">
                    <string>Сохранить</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="creditCountButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(201, 94, 22);</string>
                   </property>
                   <property name="text">
                    <string>Рассчитать</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>60</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <layout class="QFormLayout" name="creditHorizontalLayout_2">
                 <property name="verticalSpacing">
                  <number>12</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="creditSumlabel">
                   <property name="text">
                    <string>Сумма:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="creditSum"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="creditPercentlabel">
                   <property name="text">
                    <string>Годовая ставка (%):</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="creditPercent"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="creditTimelabel">
                   <property name="text">
                    <string>Срок (месяцев):</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QDateEdit" name="creditDate"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="creditDatelabel">
                   <property name="text">
                    <string>Дата выдачи:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSpinBox" name="creditTime"/>
                 </item>
                </layout>
               </item>
               <item row="4" column="0" colspan="4">
                <widget class="QTableWidget" name="creditTable">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <column>
                  <property name="text">
                   <string>Дата</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Плата в месяц</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Проценты</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Тело кредита</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Переплата</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Остаток</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="hypothectab">
           <attribute name="title">
            <string>Ипотека</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QWidget" name="hypothec" native="true">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="1" colspan="2">
                <layout class="QFormLayout" name="hypothecHorizontalLayout_8">
                 <property name="verticalSpacing">
                  <number>12</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="hypothecDatelabel">
                   <property name="text">
                    <string>Дата выдачи:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDateEdit" name="hypothecDate"/>
                 </item>
                </layout>
               </item>
               <item row="3" column="0" colspan="3">
                <widget class="QTableWidget" name="hypothecTable">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <column>
                  <property name="text">
                   <string>Дата</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Плата в месяц</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Проценты</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Тело ипотеки</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Переплата</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Остаток</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item row="1" column="0" rowspan="2" colspan="3">
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="hypothecCountButton">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(201, 94, 22);</string>
                   </property>
                   <property name="text">
                    <string>Рассчитать</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>60</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="hypothecSaveButton">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(201, 94, 22);</string>
                   </property>
                   <property name="text">
                    <string>Сохранить</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <layout class="QFormLayout" name="hypothecHorizontalLayout_4">
                 <property name="verticalSpacing">
                  <number>12</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="hypothecSumlabel">
                   <property name="text">
                    <string>Сумма:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="hypothecSum"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="hypothecFirstPaymentlabel">
                   <property name="text">
                    <string>Первоначальный взнос:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="hypothecFirstPayment"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="hypothecTimelabel">
                   <property name="text">
                    <string>Срок (лет):</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QSpinBox" name="hypothecTime"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="hypothecPercentlabel">
                   <property name="text">
                    <string>Годовая ставка (%):</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="hypothecPercent"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="deposittab">
           <attribute name="title">
            <string>Вклад</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="0" colspan="2">
             <widget class="QTableWidget" name="depositTable">
              <column>
               <property name="text">
                <string>Дата</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Начислено процентов</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Сумма</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QFormLayout" name="formLayout">
              <property name="horizontalSpacing">
               <number>9</number>
              </property>
              <property name="verticalSpacing">
               <number>12</number>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="depositSumlabel">
                <property name="text">
                 <string>Сумма вклада:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="depositSum"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="depositTimelabel">
                <property name="text">
                 <string>Срок размещения (месяцев):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="depositTime"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="depositPercentlabel">
                <property name="text">
                 <string>Годовая ставка (%):</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="depositPercent"/>
              </item>
              <item row="2" column="1">
               <widget class="QDateEdit" name="depositDate"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="depositDatelabel">
                <property name="text">
                 <string>Дата начала срока:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="depositCountButton">
                <property name="contextMenuPolicy">
                 <enum>Qt::DefaultContextMenu</enum>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(201, 94, 22);</string>
                </property>
                <property name="text">
                 <string>Рассчитать</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>60</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="database">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="2">
         <widget class="QPushButton" name="calculatorButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(114, 159, 207);</string>
          </property>
          <property name="text">
           <string>Калькулятор</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);
selection-background-color: rgb(239, 41, 41)</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_18">
           <item row="1" column="0" colspan="3">
            <widget class="QTableView" name="tableView"/>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QListWidget" name="databaseList">
             <item>
              <property name="text">
               <string>Новый элемент</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
